### PR TITLE
Issue #20972: Add traces and move to bootstrap.properties

### LINKED ISSUE #20972: Add runtime and logservice traces to com.ibm.ws.jsf.2.3_fat servers to debug application start intermittent issue
In addition to adding a new trace specification to the servers, this issue will also be used to move the trace declaration to the bootstrap.properties. We've seen a number of build breaks where some of the JSF 2.3 applications don't start-up in a reasonable amount of time as expected by the FAT framework. As such some additional traces are being enabled for all of the JSF 2.3 FAT servers to try and gather additional information.

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #20972

### CODE REVIEW

**Comment by pnicolucci** [comment]
#libby # build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#libby

File: dev/com.ibm.ws.jsf.2.3_fat/publish/servers/jsf23CDIFacesInWebXMLServer/server.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2018 IBM Corporation and others.
+    Copyright (c) 2018, 2022 IBM Corporation and others.
```
Comment (volosied) [inline dev/com.ibm.ws.jsf.2.3_fat/publish/servers/jsf23CDIFacesInWebXMLServer/server.xml:2]: This server.xml still has the logging

File: dev/com.ibm.ws.jsf.2.3_fat/publish/servers/jsf23CDIGeneralServer/server.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2017, 2018 IBM Corporation and others.
+    Copyright (c) 2017, 2022 IBM Corporation and others.
```
Comment (volosied) [inline dev/com.ibm.ws.jsf.2.3_fat/publish/servers/jsf23CDIGeneralServer/server.xml:2]: This server.xml has logging included, too

File: dev/com.ibm.ws.jsf.2.3_fat/publish/servers/jsf23CDIFacesInWebXMLServer/server.xml (hunk 1 above)
Comment (pnicolucci) [reply dev/com.ibm.ws.jsf.2.3_fat/publish/servers/jsf23CDIFacesInWebXMLServer/server.xml:2]: Good catch, fixed!

File: dev/com.ibm.ws.jsf.2.3_fat/publish/servers/jsf23CDIGeneralServer/server.xml (hunk 2 above)
Comment (pnicolucci) [reply dev/com.ibm.ws.jsf.2.3_fat/publish/servers/jsf23CDIGeneralServer/server.xml:2]: Good catch, fixed!

**Comment by pnicolucci** [comment]
#libby #build

**Review by volosied — APPROVED** [review]
LGTM

**Pull request merged**